internal/cli/server/config.go

```diff
@@ -58,9 +58,12 @@ type Config struct {
 	// GcMode selects the garbage collection mode for the trie
 	GcMode string `hcl:"gc-mode,optional"`
 
-	// XXX
+	// Snapshot enables the snapshot database mode
 	Snapshot bool `hcl:"snapshot,optional"`
```

Comment:
Why do we have both `Snapshot` and `NoSnapshot` here? Shouldn't we only have one?

Reply:
Oh yes. I never thought of this as a redundant flag. I thought this is just an additional flag.